behave_testrail_reporter/testrail_reporter.py

```diff
@@ -209,16 +209,18 @@ def _get_testrail_client(self):
 
         return self.testrail_client
 
-    def _add_test_result(self, project, case_id, status, comment='', elapsed='0'):
+    def _add_test_result(self, project, case_id, status, comment='', elapsed='1'):
```

Comment:
I'm not sure why elapsed is a string. I'd prefer a name like "elapsed_seconds" for values with implied units.

Reply:
Good point. 👍